Tools/autotest/param_metadata/mdemit.py

```diff
@@ -13,7 +13,7 @@
 
 # Parameter groups with redundant information (ie RCn_, SERVOn_)
 # We can keep the documentation concise by only documenting these once
-nparams = ['RCn_', 'SERVOn_', 'SRn_', 'BTNn_']
+nparams = ['RCn_', 'SERVOn_', 'SRn_', 'BTNn_', 'INS_TCALn_', 'BATTn_', 'BAROn_GND_', 'BAROn_WCF_', 'CAN_Dn_', 'CAN_Dn_KDE_', 'CAN_Dn_PC_', 'CAN_Dn_UC_','CAN_Dn_TST_', 'CAN_Pn_', 'RNGFNDn_']
```

Comment:
can we add `EK3_SRCn_` to this list, too?

Reply:
I’d love to, and I tried to originally, but the existing n-grouping only works for the first number, and I don’t want to dig into it enough to rewrite it, especially since it’s currently compatible with Python 2 and I don’t know if it’s allowed to use modern Python syntax instead. In general that’s a pretty big deterrent to me doing anything with ardupilot’s Python-related stuff.

I also wanted to do BARO (but the numbered ones are mixed into an unnumbered section), and COMPASS (but the numbers are after the fields - that one at least is fixed in `master` I believe)